### PR TITLE
Base centos support and data bag recipe

### PR DESCRIPTION
**Pull request by armiller**
This allows basic management of vips conf files using data bags. @osuosl-cookbooks/chefs please review as soon as possible. this is blocking the load balancers being deployed.

### CODE REVIEW

**Comment by armiller** [comment]
@osuosl-cookbooks/chefs please review.

File: recipes/default.rb [hunk 1]
```diff
@@ -17,44 +17,48 @@
...
-    :advskew => node['ucarp']['master'] ? node['ucarp']['advskew'] : node['ucarp']['advskew'] + 99,
-    :advbase => node['ucarp']['advbase'],
-    :master => node['ucarp']['master'] ? 'yes' : 'no',
-    :interface => node['ucarp']['interface'],
-    :bonded_interfaces => node['ucarp']['bonded_interfaces'],
-    :bond_mode => node['ucarp']['bond_mode']
-  )
-  notifies :restart, "service[networking]", :immediately
+unless platform_family?('rhel')
+  template "/etc/network/interfaces" do
```
Comment (jordane) [inline recipes/default.rb:25]: Do we really want to manage `/etc/network/interfaces` here? That seems dangerous and like other thigns will want to battle for it.

Comment (armiller) [inline recipes/default.rb:25]: Thats from the original cookbook. I'm only concerned right now about centos. We can fix it later.

Comment (jordane) [inline recipes/default.rb:25]: Okay. Works for me then.

**Comment by jordane** [comment]
+1

**Pull request merged**